controllers/helpers.go

```diff
@@ -508,3 +508,43 @@ func getCategoryVMSpec(client *nutanixClientV3.Client, categoryIdentifiers []*in
 	}
 	return categorySpec, nil
 }
+
+func getProjectUUID(client *nutanixClientV3.Client, projectName, projectUUID *string) (string, error) {
+	var foundProjectUUID string
+	if projectUUID == nil && projectName == nil {
+		return "", fmt.Errorf("name or uuid must be passed in order to retrieve the project")
+	}
+	if projectUUID != nil {
+		projectIntentResponse, err := client.V3.GetCluster(*projectUUID)
```

Comment:
how does GetCluster give project? OR do we need to use https://www.nutanix.dev/api_references/prism-central-v3/#/b3A6MjU1ODc2MDg-get-a-existing-project

Reply:
fix validated